sweetest/src/main/java/com/mysugr/sweetest/framework/coroutine/AssertSequence.kt

```diff
@@ -0,0 +1,15 @@
+package com.mysugr.sweetest.framework.coroutine
+
+suspend fun assertSequence(block: suspend AssertSequence.() -> Unit) {
+    block(AssertSequence())
+}
+
+class AssertSequence {
```

Comment:
after some thinking i got behind what's the purpose here, but maybe you could add a line of explanation as kdoc?